ccm_web/server/src/api/api.controller.ts

```diff
@@ -77,4 +103,26 @@ export class APIController {
     if (isAPIErrorData(result)) throw new HttpException(result, result.statusCode)
     return result
   }
+
+  @UseInterceptors(InvalidTokenInterceptor)
+  @ApiSecurity('CSRF-Token')
+  @Post('sections/:id/merge/:course')
+  async mergeSection (
```

Comment:
What do you think about 
/course/:id/mergesections
with a body of section ids?

That would allow multiple sections to be merged into the target course in a single call.

Its possible I still don't have my  head wrapped around what is  actually going on though.

Reply:
Yeah, I could do this, seems like it would simplify things for you. Does it make sense for unmerge to be bulk too?

Reply:
I have to think more about how unmerge bulk would look (like the route and parameters, body, etc.). You don't need the `courseId` for this.

Reply:
> Yeah, I could do this, seems like it would simplify things for you. Does it make sense for unmerge to be bulk too?

In the UI it looks like unmerge is intended to be section by section, but would it be just the same on the backend if it accepted an array?

Reply:
Yeah, I guess there is no reason we can't do this in a bulk way, but in practice, do we really need that functionality yet?

Reply:
@chrisrrowland, okay, are you happier with these changes? https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/199/commits/0c009a3859045bd02cafd1ef114fca6263df892b and https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/199/commits/fa17d17b2013d892f7a0f29c8299719b31cfa55e